tests/test-utils/delete-test-user.ts

```diff
@@ -0,0 +1,90 @@
+const SUPABASE_URL = process.env.NEXT_PUBLIC_SUPABASE_URL!;
+const SERVICE_ROLE_KEY = process.env.SUPABASE_SERVICE_ROLE_KEY!;
+
+if (!SUPABASE_URL || !SERVICE_ROLE_KEY) {
+  throw new Error('Set NEXT_PUBLIC_SUPABASE_URL and SUPABASE_SERVICE_ROLE_KEY in env');
+}
+
+// Safety check: ensure we're using test database
+if (!SUPABASE_URL.includes('ynexconwrqhiohlvpnav')) {
```

Comment:
What does `ynexconwrqhiohlvpnav` mean in this case? Just making sure this isn't an id that shouldn't be in the code 😅